docs/howto/openstack/neutron/create-security-groups.md

```diff
@@ -0,0 +1,55 @@
+# Creating security groups
+
+[By definition](https://docs.openstack.org/nova/latest/admin/security-groups.html), security groups are "[...] _sets of IP filter rules that are applied to all project instances, which define networking access to the instance. Group rules are project specific; project members can edit the default rules for their group and add new rule sets_".
```

Comment:
Micro-nit about punctuation: the whole quote (including the quotation marks) should be in italics, and the full-stop needs to be within the quotation marks because English is weird. Thus:

```markdown
security groups are _"[...] sets of IP filter rules that are applied to all project instances,
which define networking access to the instance. Group rules are project specific;
project members can edit the default rules for their group and add new rule sets."_
```

Reply:
I never ever know the correct way with punctuation and quotation marks, thanks! Fixed.